orders/models.py

```diff
@@ -0,0 +1,86 @@
+from django.db import models
+from django.utils import timezone
+from daycare.models import DayCare
+from dogowner.models import DogOwner
+
+
+class StatusOptions(models.TextChoices):
+    Pending = 'P', 'Pending'
+    Approved = 'A', 'Approved'
+    Canceled = 'C', 'Canceled'
+    Finished = 'F', 'Finished'
+    OnGoing = 'O', 'On Going'
+
+
+class Order(models.Model):
+    order_id = models.AutoField(primary_key=True)
+    dog_owner_id = models.ForeignKey(DogOwner, on_delete=models.DO_NOTHING)
+    daycare_id = models.ForeignKey(DayCare, on_delete=models.DO_NOTHING)
+    booking_date = models.DateTimeField(auto_now=True)
+    approving_date = models.DateTimeField(blank=True, null=True)
+    starting_date = models.DateTimeField()
+    finishing_date = models.DateTimeField()
```

Comment:
I think better approach here should be saving `duration` instead of `finishing_date` and the date will be calculated from the `starting_date` + `duration` for exaple:
```
import datetime as DT
today = DT.date.today()
week_ago = today + DT.timedelta(days=duration)
```

Reply:
> I think better approach here should be saving `duration` instead of `finishing_date` and the date will be calculated from the `starting_date` + `duration` for exaple:
> 
> ```
> import datetime as DT
> today = DT.date.today()
> week_ago = today + DT.timedelta(days=duration)
> ```

The idea comes from the need of importing those dates into the dog owner's and daycare's calendars, and because the dog owner chooses the starting and finishing date of an order, just like Airbnb or Booking.
So converting it from DateTime to duration, and then from duration back to DateTime feels like a waste.
Are you agree?

Reply:
In the calendars scenario I agree to put the data in calendar form its easier to have two DateTimes but what about reports? for example calculating the total price is much easier with duration

I think that there are pros and cons for both ways let's see what others think

Reply:
Exactly what I was thinking.
Plus, we can calculate the total price per night, so it would be easier